src/java/main/org/apache/zookeeper/server/NettyServerCnxnFactory.java

```diff
@@ -170,7 +185,6 @@ public void messageReceived(ChannelHandlerContext ctx, MessageEvent e)
                 throw ex;
             }
         }
-
```

Comment:
Ops, again Flavio's previous review comment: https://github.com/apache/zookeeper/pull/77/files#r80004333 (should have put referenced this here).